### PR TITLE
Fix #141901 www.tf1info.fr

### LINKED ISSUE #141901: www.tf1info.fr
### Issue URL (Incorrect Blocking)
 [https://www.tf1info.fr/direct/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.tf1info.fr%2Fdirect%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.12
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Block WebRTC, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Block Java
DNS filtering: | server: `https://family.cloudflare-dns.com/dns-query`<br>filters: `https://filters.adtidy.org/windows/filters/15.txt, User+rules`
Filters: | <b>Ad Blocking:</b><br/>EasyList, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>Peter Lowe's Blocklist, <br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking, <br/>Legitimate URL Shortener<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/>Online Malicious URL Blocklist, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>Liste FR, <br/>AdGuard French, <br/>AdGuard Russian
Userscripts: | https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js,<br>https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/adguard-wot/1.0/wot.user.js

### PR DESCRIPTION
**Pull request by contribucious**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

Closes #141901

:arrow_right_hook: More precisely, fixing https://github.com/AdguardTeam/AdguardFilters/issues/141901#issuecomment-1433076690 — a rare timing problem case IMHO, and only concerning people using blocking at the DNS level, typically.

### Add your comment and screenshots

Please see posts `Changelog` and `Remark(s)` below.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by contribucious** [comment]
# Changelog
&nbsp;
### :heavy_plus_sign: Added to fix the issue

```adblock
!
! In FrenchFilter > antiadblock.txt file:
!

! https://github.com/AdguardTeam/AdguardFilters/issues/141901#issuecomment-1434658668
tf1.fr,tf1info.fr#%#//scriptlet('prevent-fetch', 'static.adsafeprotected.com/favicon.ico method:HEAD')
tf1.fr,tf1info.fr#%#//scriptlet('prevent-fetch', '/^https:\/\/ads\.stickyadstv\.com\/$/ method:HEAD')
```

### :heavy_plus_sign: Added as bonus _(detailed comments only present here)_

```adblock
!
! In FrenchFilter > antiadblock.txt file:
!

! Other 'set-constant' stuff will prevent the AAB check request to be done,
! while this one will ensure that even if done, the result doesn't matter.
tf1.fr#%#//scriptlet('set-constant', '__TF1_CONFIG__.featureFlag.contentAccess.isAdblockCheckRequired', 'false')
```

```adblock
!
! In FrenchFilter > general_extensions.txt file:
!

! Additional disabling of video ads (tf1info.fr side)
! in addition to the generic player iframe itself stuff (just as an extra precaution).
tf1info.fr#%#//scriptlet('set-constant', '__NEXT_DATA__.runtimeConfig.playerTF1.ads.enable', 'false')
```

### :magic_wand: Changed as bonus

- **[tf1.fr, tf1info.fr]** Did the JS rules → Scriptlets migration.
  _(Thus, improving the user experience of uBlock Origin users on these popular websites.)_

- **[tf1.fr]** Fixed the rule to hide the bottom banner (about the paid "MYTF1 MAX" offer).

**Comment by contribucious** [comment]
# Remark(s)
&nbsp;
:thought_balloon: Not sure if these two lines of comment are still required nowadays, by the way:
https://github.com/AdguardTeam/AdguardFilters/blob/b058daad9339b76320404d9af0ea6de20cee8390/FrenchFilter/sections/antiadblock.txt#L573-L574

File: FrenchFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -576,10 +579,13 @@ prod-player.tf1.fr,tf1.fr,tf1info.fr#@#.carbon-ads
...
-! (2/2) "NETWORK REQUEST" AAB CHECK
+! (3/3) "NETWORK REQUEST" AAB CHECK
 !
-tf1.fr#%#AG_defineProperty('__TF1_CONFIG__.adblock.serverRequest', { value: false, writable: false });
-tf1info.fr#%#AG_defineProperty('__NEXT_DATA__.runtimeConfig.adBlock.serverRequest', { value: false, writable: false });
+tf1.fr#%#//scriptlet('set-constant', '__TF1_CONFIG__.adblock.serverRequest', 'false')
+tf1info.fr#%#//scriptlet('set-constant', '__NEXT_DATA__.runtimeConfig.adBlock.serverRequest', 'false')
+! https://github.com/AdguardTeam/AdguardFilters/issues/141901#issuecomment-1434658668
+tf1.fr,tf1info.fr#%#//scriptlet('prevent-fetch', 'static.adsafeprotected.com/favicon.ico method:HEAD')
+tf1.fr,tf1info.fr#%#//scriptlet('prevent-fetch', '/^https:\/\/ads\.stickyadstv\.com\/$/ method:HEAD')
```
Comment (Alex-302) [inline FrenchFilter/sections/antiadblock.txt:588]: is `method` required here?

**Review by Alex-302 — APPROVED** [review]

File: FrenchFilter/sections/antiadblock.txt (hunk 1 above)
Comment (contribucious) [reply FrenchFilter/sections/antiadblock.txt:588]: Yes, ideally. I mean that, in all cases (even for the `favicon.ico` request, see for example [here for tf1info.fr](https://user-images.githubusercontent.com/4764956/220398649-f3b3bbba-d978-4235-ad67-42806cef726f.png)), they only do HEAD requests for AAB checks:
- [tf1.fr](https://user-images.githubusercontent.com/4764956/220392899-f6f6a23e-1a57-4eda-a7dc-ab8c1a507095.png)
- [tf1info.fr](https://user-images.githubusercontent.com/4764956/220392946-c215b44f-0d6d-465c-b575-d637dd45985e.png)

_(+ confirmed in Firefox "Developer Tools" > "Network" tab as shown above and all tested. :thumbsup:)_

Comment (Alex-302) [reply FrenchFilter/sections/antiadblock.txt:588]: Ok, thanks.

Comment (contribucious) [reply FrenchFilter/sections/antiadblock.txt:588]: No problem. Thank _you_! 🙂

**Pull request merged**